main/sqlite/APKBUILD

```diff
@@ -48,8 +47,6 @@ _amalgamation="-DSQLITE_ENABLE_FTS4 \
 	-DSQLITE_ENABLE_JSON1"
 
 build() {
-	cd "$builddir"
-
```

Comment:
I dont like the removal of `cd "$builddir"` because it may mess up things when i backport to older stable branches for security reasons. Its not that it is not possible for me to re add them when backporting, it just slows me down a bit.

Reply:
Reverted.